console/ut/console.c

```diff
@@ -449,11 +446,10 @@ static void cons_client_fini(struct m0_rpc_client_ctx *cctx)
 
 static void cons_server_init(struct m0_rpc_server_ctx *sctx)
 {
-	int result;
-
+	int                 result;
```

Comment:
No need to change the variable alignment.

Reply:
incorporated